monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+### _Les Signatures de Cercle_
+
+Pour assurer la confidentialité des émetteurs, Monero fait plus que garder le secret des adresses. L’ensemble de la base des données de Monero existe sur sa chaine de blocs sous forme de transactions sortantes dénommées TXO. À chaque TXO unique est assigné une identité unique (ID), appelée clé publique (également nommée "adresse furtive"). Par simple analyse de la chaine de blocs de Monero, il est impossible d'établir une corrélation entre les clés publiques des TXO et les adresses des portefeuilles électroniques de Monero. Cela étant, les clés publiques ont également besoin d’une protection. Il serait inacceptable que les flux de Monero parmi les clés publiques soient transparents, considérant que des techniques d'analyses externes pourraient associer ces clés publiques a des portefeuilles, puis à des individus.
+
+Les flux de Monero des clés publiques sont gardés confidentiels par l’usage d’un procédé de chiffrage nommé "cryptographic ring signatures". Ce procédé permet de dissimuler la clé publique d’un TXO en l'incorporant à un groupe de transaction-leurres (également nommée "mixins"). L’observation d’une transaction ne permettra donc pas de déterminer si un TXO donné est l’original ou non. Vous pouvez voir un exemple d’émission ou de réception de TXO en cherchant une transaction a l’aide d’un programme d’analyse de la chaine de blocs de Monero tel que [moneroblocks.info](https://moneroblocks.info/) ou [xmrchain.net](https://xmrchain.net/).
+
+La manière dont les transaction-leurres TXO sont sélectionnées a partir de la chaine de blocs s’est améliorée au fil du temps. Au tout début, l'intégration de transaction-leurres - bien que faisant partie du système - restait facultative, et les leurres étaient sélectionnées uniformément sur l'historique entière des transactions. Avec le temps, le nombre de transaction-leurres grandit et devint obligatoire, pour atteindre aujourd'hui exactement 10 transaction-leurres par ordre d’exécution. De même, le procédé statistique de sélection fut affiné au fil du temps pour lui donner plus de complexité grâce au "tailored random process" (un algorithme basé sur le procédé de sélection pour l’exécution de l’ordre d’après une théorie statistique de distribution gamma particulière, que nous utilisons encore a ce jour).
+
+Les algorithmes du "core ring signature" ont également été améliorés: la publication de proposition originelle ("whitepaper") de CryptoNote présentait un processus de signature de cercle différent de celui que nous employons actuellement, qui est bâti sur l’utilisation des signatures "Multilayered Linkable Spontaneous Anonymous Group" (MLSAG). Une amélioration substantielle, nommée signatures "Compact Linkable Spontaneous Anonymous Group" (CLSAG), est attendue prochainement, et promet de réduire d’environ un quart le poids (en termes de données) des transactions et d’un dixième leur temps de confirmation (pour une transaction de deux entrées et deux sorties).
```

Comment:
1.
Original sentence:
Les algorithmes du "core ring signature" ont également été améliorés
Suggested sentence:
Le coeur des algorithmes a signature de cercle a également été amélioré   

Reason: The translation of 'ring signature' was not performed. Furthermore, the meaning of 'core ring signature algorithms' is slightly ambiguous in the original english text (due to a lack of conjunctions), but I think my suggested translation is what the original authors meant.

2.
signature de cercle différent de celui que nous employons actuellement  >>> signature de cercle moins efficace et différent de celui que nous employons actuellement 

Reason: The notion of 'more efficient' regarding the new algorithm was missing in the translation

Reply:
Good point on both counts. Integrated,

Reply:
Can you change "Le coeur des algorithmes a signature de cercle a également été amélioré" to "Le coeur des algorithmes à signature de cercle a également été amélioré" ?

My correction had a typo (a instead of à), which I missed (my bad!).

Reply:
Integrated, good catch!